package.json

```diff
@@ -30,6 +30,8 @@
     "bookshelf": "^1.2.0",
     "hapi": "16.6.1",
     "hapi-bookshelf-serializer": "2.1.0",
+    "hapi-format-error": "^3.0.0",
+    "joi": "12.0.0",
```

Comment:
didn't know we're still running Joi version 12 for the project. In that case, my comment above might not be valid, since I tested it on Joi version 17.